.circleci/config.yml

```diff
@@ -76,6 +76,21 @@ jobs:
             - "~/.ivy2/cache"
             - "~/.sbt"
             - "~/.m2"
+  test213_jdk17:
```

Comment:
A bit concerned about the CI time and cost if we test all combinations 🤔 Shall we test JDK 11? OR maybe 2.13 with 11 and 3 with 17?

Reply:
I think CI time will be mostly unaffected since the workflows are run in parallel. But I agree about cost.

The issue I see with testing Scala 2.13 with JDK11 and Scala 3 with JDK17 is that if there's an issue with the Scala 3 test suite we won't know whether it's cause of JDK17 or Scala 3.

Perhaps we can test Scala 3 against JDK 11 & 17 and leave Scala 2.13 tested just against 11?

Reply:
I checked our CI concurrency limit (30) and credits (we use about 25%) so there's some margin. Maybe it's okay for now but it's going to be worse when we add JDK 21 so I wonder if there is a better way. Maybe we could do it on `series/2.x` only which means we find it after merging, but at least before doing a release.

Reply:
Yeah I think that's a good tradeoff. Also _in case_ something breaks, it doesn't block other PRs from getting merged